website/docs/r/privateca_certificate_authority.html.markdown

```diff
@@ -134,7 +134,7 @@ resource "google_project_service_identity" "privateca_sa" {
   service  = "privateca.googleapis.com"
 }
 
-resource "google_kms_crypto_key_iam_binding" "privateca_sa_keyuser" {
+resource "google_kms_crypto_key_iam_binding" "privateca_sa_keyuser_signerverifier" {
```

Comment:
If you change the TF resource name, you also need to change the depends_on below.

Reply:
I see its fixed on the upstream PR, so please ignore this. :)